osu.Game/Screens/OnlinePlay/Lounge/Components/PlaylistCountPill.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Specialized;
+using Humanizer;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+
+namespace osu.Game.Screens.OnlinePlay.Lounge.Components
+{
+    /// <summary>
+    /// A pill that displays the playlist item count.
+    /// </summary>
+    public class PlaylistCountPill : OnlinePlayComposite
+    {
+        private OsuTextFlowContainer count;
+
+        public PlaylistCountPill()
+        {
+            AutoSizeAxes = Axes.Both;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            InternalChild = new PillContainer
+            {
+                Child = count = new OsuTextFlowContainer(s => s.Font = OsuFont.GetFont(size: 12))
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                }
```

Comment:
Another thing I've missed to mention from my previous review is that currently centered text in pills get misaligned vertically:
![CleanShot 2021-08-05 at 17 44 45](https://user-images.githubusercontent.com/22781491/128369891-8cb65509-5af0-4b8e-ad81-6ba43f63543c.png)

In the figma designs, they're centered with a bottom padding of 1-2 depending on the font size:
![CleanShot 2021-08-05 at 17 46 05](https://user-images.githubusercontent.com/22781491/128370131-44082b0f-b9e7-43b0-a418-d16fa1b3f23a.png)

Should padding also be applied in the game, in a way that it becomes centered? or is that supposed to be handled somewhere framework-side?

Pointing this out because I also faced this during my work on the beatmap info overlay redesign, and had to apply `Padding` to not make the overlay pills look silly.

Reply:
Looks like the figma padding should be matched in this case.